give-activecampaign.php

```diff
@@ -431,9 +431,10 @@ public function subscribe_email( $user_info = [], $lists = [], $tags = [] ) {
 					"email"      => $user_info['email'],
 					"first_name" => $user_info['first_name'],
 					"last_name"  => $user_info['last_name'],
-					"tags"       => implode( ', ', $tags ),
 				];
 
+                is_array($tags) && $subscriber["tags" ] = implode( ', ', $tags );
+
```

Comment:
@JoshuaHungDinh this is a good start identifying what the issue is.  However, with types it's better to fix the issue up the chain so the function can focus on logic. Ideally for this, we would update the function params with types and cast the values to arrays during implementation.

I realize that might be too heavy handed to fix a legacy issue right now so how about we cast the `$tags` value instead so the logic remains consistent with how it was originally? 

```
tags" => implode( ', ', (array)$tags ),
```

Reply:
Sounds good, I'll keep that in mind. Thanks! aa09a607ffb5ce81472fce61009b1476ccbcb960
Also the file spacing/format looks a little off to me. In my ide everything is lined up though. Does it look ok on your end?

Reply:
Thanks! yes but looks okay in my ide so all good